README.md

```diff
@@ -1,5 +1,88 @@
 # Skywalking Grafana Plugins
 
+SkyWalking Grafana Plugins provides an extention for [Apache SkyWalking](https://skywalking.apache.org/) visualizting telemetry data on Grafana. 
+
+Currently, SkyWalking supports [PromQL](https://skywalking.apache.org/docs/main/next/en/api/promql-service/) to establish [Grafana metrics dashboard](https://skywalking.apache.org/docs/main/next/en/setup/backend/ui-grafana/). This plugin is focusing on more telemetry data, Service Map Topology.
+
+This plugin provides a Grafana data source implementation.
+
+## Install the latest version of SkyWalking data source in your Grafana application
+
```

Comment:
We should set the Grafana version(lowest at least)

Reply:
Yes, current version is 9.5.1.

Reply:
Which version do we test? How about all 9.x?

Reply:
```suggestion

* Require Grafana 9.5.1

```